<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>-1</y>
      <width>451</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="strona_1">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>451</width>
        <height>491</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="accessibleName">
          <string>label</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="next_button">
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="strona_2">
     <widget class="QLabel" name="testowa">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Witaj swiecie!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>420</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>130</y>
        <width>67</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="radio_arch_32">
         <property name="text">
          <string>&amp;32 Bit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio_arch_64">
         <property name="text">
          <string>&amp;64 Bit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="strona_3">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>156</width>
        <height>239</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QCheckBox" name="checkSoft_1">
         <property name="text">
          <string>Sterownik Synaptics</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkSoft_2">
         <property name="text">
          <string>Libre Office</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkSoft_3">
         <property name="text">
          <string>Dropbox</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkSoft_4">
         <property name="text">
          <string>Audacious</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkSoft_5">
         <property name="text">
          <string>VLC Player</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkSoft_6">
         <property name="text">
          <string>Pidgin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkSoft_7">
         <property name="text">
          <string>Steam</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkSoft_8">
         <property name="text">
          <string>Skype</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkSoft_9">
         <property name="text">
          <string>Gimp</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>400</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>60</y>
        <width>134</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="radio_lxdm">
         <property name="text">
          <string>Menedzer lo&amp;gowania</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio_autolog">
         <property name="text">
          <string>Auto&amp;logowanie</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="strona_4">
     <widget class="QCheckBox" name="check_home">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>150</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Parytcja /home</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>70</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Partycja /</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="check_swap">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>240</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Partycja swap</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>410</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_root">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>90</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_home">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>180</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_swap">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>270</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="strona_5">
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>390</y>
        <width>151</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nazwa uzytkownika:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_user">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>49</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Haslo:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_user_password">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Haslo root:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_root_password">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="strona_6">
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>230</y>
        <width>271</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>check_home</sender>
   <signal>toggled(bool)</signal>
   <receiver>line_home</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_swap</sender>
   <signal>toggled(bool)</signal>
   <receiver>line_swap</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
